src/Enigma2.cpp

```diff
@@ -1,6 +1,6 @@
 /*
- *      Copyright (C) 2005-2019 Team XBMC
- *      http://xbmc.org
+ *      Copyright (C) 2005-2020 Team Kodi
+ *      http://kodi.tv
```

Comment:
This should be `https` everywhere!

Reply:
Ah OK, becomes changed, was gone down a bit by the round.

Reply:
Have changed to `https`.